images/inference-services/ovms/run.sh

```diff
@@ -60,7 +64,7 @@ resources: ${RESOURCES}
 target_device: "${FUSEML_TARGET_DEVICE}"
 EOF
 
-cat /opt/openvino/templates/values.yaml
+$FUSEML_VERBOSE && cat /opt/openvino/templates/values.yaml
```

Comment:
don't you want to add this to kfserving and seldon too?

Reply:
done